resources/js/lang/en.js

```diff
@@ -22,9 +22,9 @@ export default {
         2: {
             "name": "The Merchant's Wrath",
             "stages": {
-                0: "A tribe of Inox in the Dagger Forest has ransacked a couple of Jekserah's caravans headed to the Capital. If you can make an example of them, she will pay you handsomely.",
-                1: "A tribe of Inox in the Dagger Forest has ransacked a couple of Jekserah's caravans headed to the Capital. If you can make an example of them, she will pay you handsomely.{br}The Inox tribe is dead. You've made an brutal example of them and no one will threaten Jekserah's caravans for quite a while. You don't exactly feel good about slaughtering parents in front of their children, but Jekserah paid you all the same.",
-                2: "A tribe of Inox in the Dagger Forest has ransacked a couple of Jekserah's caravans headed to the Capital. If you can make an example of them, she will pay you handsomely.{br}Jekserah has fled from Gloomhaven and sworn vengeange upon you due to your actions against her. She will not be keen on paying you for running her errands anymore."
+                0: "Jekserah offers you another job after you finished your first one. A tribe of Inox in the Dagger Forest has ransacked a couple of her caravans headed to the Capital. If you can make an example of them, she will pay you even more this time.",
+                1: "Jekserah offered you another job after you finished your first one. A tribe of Inox in the Dagger Forest had ransacked a couple of her caravans headed to the Capital. She asked if you could make an example of them.{br}The Inox tribe is dead. You've made a brutal example of them and no one will threaten Jekserah's caravans for quite a while. You don't exactly feel good about slaughtering parents in front of their children, but Jekserah paid you all the same.",
+                2: "Jekserah offered you another job after you finished your first one. A tribe of Inox in the Dagger Forest had ransacked a couple of her caravans headed to the Capital. She asked if you could make an example of them.{br}Jekserah has fled from Gloomhaven and sworn vengeance upon you due to your actions against her. She will not be keen on paying you for running her errands anymore."
```

Comment:
These edits (as well as the change in quest 1 section 2) don't seem like an improvement to me, except for correcting the vengeance typo.

Reply:
I felt like it was missing some info about the 'preceding events'. And I tried to stay close to the original text

>If you’re interested, I may have another job for you. A tribe of Inox in the Dagger Forest has ransacked a couple of my caravans headed to the Capital.

I could change

>she will pay you more this time

and remove 'this time' because that isn't in the original.